modules/Core/pages/profile.php

```diff
@@ -711,10 +710,9 @@
             }
             $replies_query = null;
 
-            $target_user = new User($post_user[0]->id);
             $wall_posts[] = [
                 'id' => $nValue->id,
-                'user_id' => Output::getClean($post_user[0]->id),
+                'user_id' => $target_user->data()->id,
```

Comment:
Where is $target_user defined if not on line 714 anymore?

Reply:
Line 639!